amethyst/src/main/java/com/vitorpamplona/amethyst/model/LocalCache.kt

```diff
@@ -1845,6 +1845,21 @@ object LocalCache {
         )
     }
 
+    /**
+     * Will return true if supplied note is one of events to be excluded from
+     * search results.
+     */
+    private fun excludeNoteEventFromSearchResults(note: Note): Boolean =
+        (
+            note.event is GenericRepostEvent ||
+                note.event is RepostEvent ||
```

Comment:
Hmm, not sure why these two need to be excluded. Maybe we could add a check for if the repost event is also one of the others we are excluding?(like FileHeaderEvent)

Reply:
Got it.